checker/src/main/java/org/checkerframework/checker/mustcall/MustCallAnalysis.java

```diff
@@ -0,0 +1,59 @@
+package org.checkerframework.checker.mustcall;
+
+import com.google.common.collect.ImmutableSet;
+import com.sun.tools.javac.code.Type;
+import java.util.HashSet;
+import java.util.Set;
+import javax.lang.model.type.TypeKind;
+import javax.lang.model.type.TypeMirror;
+import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.framework.flow.CFAnalysis;
+
+/**
+ * The analysis for the Must Call Checker. The analysis is specialized to ignore certain exception
+ * types; see {@link #isIgnoredExceptionType(TypeMirror)}.
+ */
+public class MustCallAnalysis extends CFAnalysis {
+
+  /**
+   * Constructs an MustCallAnalysis.
+   *
+   * @param checker the checker
+   * @param factory the type factory
+   */
+  public MustCallAnalysis(BaseTypeChecker checker, MustCallAnnotatedTypeFactory factory) {
+    super(checker, factory);
+  }
+
+  /**
+   * The fully-qualified names of the exception types that are ignored by this checker when
+   * computing dataflow stores.
+   */
+  protected static final Set<String> ignoredExceptionTypes =
```

Comment:
Not a full review but a quick question. Does this need to respect the new setting around ignored exceptions that were added recently?